changelogs/changelog_2_0_0.md

```diff
@@ -260,6 +260,8 @@
 
 - `strutils.split` and `strutils.rsplit` now forbid an empty separator.
 
+- Relative imports will not resolve to stdlib anymore, e.g. `import ./tables` now reports an error properly.
```

Comment:
wait .. only `stdlib` or also `-p:path`? ie the latter should also not be searched

Reply:
Yeah, make sense, I will correct the changelog.